File: Models/SmallProject.Models/OfferModel.cs
using System;

namespace SmallProject.Models
{
    public class OfferModel
    {
        public Guid Id { get; set; }

        public int Distance { get; set; }

        public int LivingArea { get; set; }

        public int AtticArea { get; set; }

        public bool PianoIncluded { get; set; }

        public Guid UserId { get; set; }

        public int TotalPrice { get; set; }
    }

    public class CreateOfferModel
    {
        public int Distance { get; set; }

        public int LivingArea { get; set; }

        public int AtticArea { get; set; }

        public bool PianoIncluded { get; set; }
    }

    public class DetailsOfferModel
    {
        public Guid Id { get; set; }

        public int Distance { get; set; }

        public int LivingArea { get; set; }

        public int AtticArea { get; set; }

        public bool PianoIncluded { get; set; }

        public string UserFullName { get; set; }

        public int TotalPrice { get; set; }
    }
}

File: UI/SmallProject.UI/ClientApp/src/app/shared/models/offer.model.ts
export class OfferModel{
    public id: string = '';
    public distance: number = 0;
    public livingArea: number = 0;
    public atticArea: number = 0;
    public pianoIncluded: boolean = false;
    public userId: string = null;
    public totalPrice: number = 0;

    constructor(){

    }
}

export class GetOfferModel{
    public Distance: number = 0;
    public LivingArea: number = 0;
    public AtticArea: number = 0;
    public PianoIncluded: boolean = false;

    constructor(){
        
    }
}

export class DetailsOfferModel{
    public id: string = '';
    public distance: number = 0;
    public livingArea: number = 0;
    public atticArea: number = 0;
    public pianoIncluded: boolean = false;
    public userFullName: string = '';
    public totalPrice: number = 0;

    constructor(){}
}